robottelo/performance/thread.py

```diff
@@ -60,11 +67,33 @@ def run(self):
 
 
 class SubscribeAttachThread(PerformanceThread):
-    def __init__(self, thread_id, thread_name, time_result_dict,
-                 num_iterations, sub_id, default_org, environment, vm_ip):
+    """Thread for Subscription by register and attach
+
+    separate `time_result_dictionary` into two new dictionaries:
+    time_result_dict_register, containing timing results of register step
+    time_result_dict_attach, containing timing results of attach step
+
+    The data structure of dictionaries now is:
+    dict-register: {client-0: [...], ..., client-9:[...]}
+    dict-attach: {client-0: [...], ..., client-9:[...]}
+
+    """
+    def __init__(
+            self,
+            thread_id,
+            thread_name,
+            time_result_dict_register,
```

Comment:
@Ichimonji10 So, since time_result_dict has been split into two separate lists, (as denoted by new data structure in docstrings), I would invoke this constructor as:
`thread = SubscribeAttachThread(`
`                i,`
`                thread_name,`
`                time_result_dict={},`
`                time_result_dict_register,`
`                time_result_dict_attach,`
`                self.num_iterations,`
`                ...`
`            )`